dali/pipeline/operator/op_schema.cc

```diff
@@ -293,4 +292,8 @@ bool OpSchema::IsTensorArgument(const std::string &name) const {
   return ret;
 }
 
+bool OpSchema::ArgSupportsPerFrameInput(const std::string &arg_name) const {
+  return per_frame_arguments_.find(arg_name) != per_frame_arguments_.end();
```

Comment:
Yep, it's done on purpose, that's how I understood @mzient suggestion to make per-frame handling explicit and not inherit it. Otherwise I would merge `per_frame_arguments_` and `tensor_arguments_` into `map<string, TensorArgDesc>`.

Reply:
That was a misunderstanding. By "explicit" I only meant that a sequence operator should require an argument to be explicitly marked for expansion and not try to expand all arguments with suitable layout.

Reply:
Sure, changed it so that the per-frameness support is part of tensor argument definition and thus inherited.